masonry/src/widgets/label.rs

```diff
@@ -386,7 +386,7 @@ impl Widget for Label {
         };
         if self.alignment_changed {
             self.text_layout
-                .align(Some(alignment_width), self.alignment);
+                .align(Some(alignment_width), self.alignment, false);
```

Comment:
Obviously I'm not super happy about an ugly boolean parameter appearing here. For reference, it seems to be from https://github.com/linebender/parley/pull/241, meaning whether we align when overflowing.
The lack of tests in that PR to make it more comprehensible isn't ideal...

In this case:
1) I agree that `false` maintains the same behaviour here. I think that true would maybe be a better default, but that should be a follow-up
2) I think that https://github.com/linebender/parley/issues/247 should block parley 0.3 (cc @tomcur); the fact that every use in https://github.com/linebender/parley/pull/241 needed to be documented with the parameter name as a comment strongly supports this...

Reply:
> I think that https://github.com/linebender/parley/issues/247 should block parley 0.3

Yes, that issue is blocking 0.3. Nico opened a PR fixing it yesterday: https://github.com/linebender/parley/pull/278.

> I agree that false maintains the same behaviour here. I think that true would maybe be a better default, but that should be a follow-up

I believe `false` (start-alignment for lines that overflow) is what web does, and currently that's the planned default for the `AlignmentOptions` introduced in Nico's PR.